bonsai/Views/FlexibleView/FlexibleView.swift

```diff
@@ -0,0 +1,32 @@
+// Thx
+// https://github.com/FiveStarsBlog/CodeSamples/tree/48e493a2b4acd7196c176689a8f3038936f0ed41/Flexible-SwiftUI
+
+import SwiftUI
+
+/// Facade of our view, its main responsibility is to get the available width
+/// and pass it down to the real implementation, `_FlexibleView`.
```

Comment:
At the first glance I don't see any reasons to divide this view into two classes. I'll try to combine them myself and figure out why they're divided